microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Prerelease testing using preview channels
+description: How to test your WebView2 app by using the preview channels of Microsoft Edge (Edge Canary, Edge Dev, or Edge Beta) to find issues that will affect your particular WebView2 app, before WebView2 changes migrate to the Evergreen WebView2 Runtime.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 02/15/2024
+---
+# Prerelease testing using preview channels
+
+_Prerelease testing_ means testing your WebView2 app by using the preview channels of Microsoft Edge to find issues that will affect your particular WebView2 app, before WebView2 changes migrate to Evergreen WebView2 Runtime.  Prerelease testing helps your app catch bugs in prerelease channels of WebView2, before the WebView2 changes ship in a stable release and degrade the app experience for end users.  In prerelease testing, you validate your app against early builds of WebView2, to ensure that your app will work with the upcoming changes.
```

Comment:
I have not seen such phrasing used: `changes migrate to Evergreen WebView2 Runtime`. I would say something like `before changes will reach Stable channel of wv2 evergreen runtime.`

Reply:
The current latest wording:

_Prerelease testing_ means testing your WebView2 app by using the preview channels of Microsoft Edge to find issues that will affect your particular WebView2 app, **before any changes ship in the Stable Evergreen Runtime**.  Prerelease testing helps your app catch bugs in prerelease channels of WebView2, **before the WebView2 changes ship in a stable release** and degrade the app experience for end users.  In prerelease testing, you validate your app against early builds of WebView2, to ensure that your app will work with the upcoming changes.